lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -254,6 +255,42 @@ def revision(scm, target, ref)
   end
 end
 
+desc "Rebuild .fixture.yml from metadata.json file"
+task :fixture_rebuild do
+  begin
+    fixtures = YAML.load_file('.fixtures.yml')['fixtures']
+  rescue Errno::ENOENT
+    fail "Unable to load .fixtures.yml"
+  end
+  
+  begin
+    metadata = JSON.parse(File.read('metadata.json'))
+  rescue Errno::ENOENT
+    fail ".fixtures.yml does not exist"
+  rescue Errno::EACCES
+    fail "Unable to read .fixtures.yml"
```

Comment:
Ah. sorry, I just looked at the code again and it hit me like duh!... OK fair enough....